TZStackView/TZStackView.m

```diff
@@ -0,0 +1,92 @@
+//
+//  TZStackView.m
+//  TZStackView
+//
+//  Created by Kevin Wooten on 10/16/15.
+//  Copyright © 2015 Tom van Zummeren. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <objc/runtime.h>
+
+
+@interface TZModuleLocator : NSObject
+@end
+
+@implementation TZModuleLocator
+@end
+
+
+// ----------------------------------------------------
+// Runtime injection start.
+// Assemble codes below are based on:
+// https://github.com/0xced/NSUUID/blob/master/NSUUID.m
+// ----------------------------------------------------
+
+#pragma mark - Runtime Injection
+
+__asm(
+      ".section        __DATA,__objc_classrefs,regular,no_dead_strip\n"
+#if	TARGET_RT_64_BIT
+      ".align          3\n"
+      "L_OBJC_CLASS_UIStackView:\n"
+      ".quad           _OBJC_CLASS_$_UIStackView\n"
+#else
+      ".align          2\n"
+      "_OBJC_CLASS_UIStackView:\n"
+      ".long           _OBJC_CLASS_$_UIStackView\n"
+#endif
+      ".weak_reference _OBJC_CLASS_$_UIStackView\n"
+      );
+
```

Comment:
Declares a weakly linked (aka if available) reference to the UIStackView class definition structure. It's used below to insert our UIStackView into the system class table.